data/hai/hai reveal 1 intro.txt

```diff
@@ -177,15 +178,15 @@ mission "ZZ Event Hai Secret Leaks: RSHU"
 		event "news: hai video clip published"
 		conversation "hai reveal republic or syndicate news"
 	on visit
-		dialog "You should first visit Allhome, where the most humans are, to see how things are going before you get more heavily involved."
+		dialog "You should first visit Allhome, where most of the humans are in Hai space, to see how things are going before you get more heavily involved."
```

Comment:
```suggestion
		dialog "You should first visit Allhome, where most of the humans in Hai space congregate, to see how things are going before you get more heavily involved."
```

"Live" would work as well, if not better, if it's accurate, but I can't recall if that's so. "Are" is just so poorly vague anyway, but also repeated in such a short time.

Reply:
Allhome should be the Hai planet with the most humans, because there are supposed to be small cities with populated by humans near the spaceport, and it was settled jointly by humans and Hai. Greenwater only has "scattered" human fishing villages but still a "sizable" human population.